docs/tutorial/get-started.md

```diff
@@ -55,14 +55,13 @@ If you want to learn about Scylla, please refer to [the official website](https:
     Deploy Scylla as a backup database.
 
     ```bash
-    kubectl apply -f k8s/jobs/db/initialize/cassandra/configmap.yaml
-    kubectl apply -f k8s/external/scylla
+    make k8s/external/scylla/deploy
```

Comment:
Is it okay that users use Makefile tasks? :thinking:

Reply:
how do you think?
I think scylla is not core value of this document, imo it's okay.

Reply:
I think the users who are reading Get Started document should know what is done in the example script. So it is better to write **exact** commands in the document. I think the former document is better because Makefile hides the exact commands.

Reply:
should we describe about scylla's exact commands??

Reply:
I added more description about this section, thank you for your notice.